x-pack/legacy/plugins/lens/public/indexpattern_plugin/state_helpers.ts

```diff
@@ -10,7 +10,7 @@ import { operationDefinitionMap, IndexPatternColumn } from './operations';
 import { IndexPattern, IndexPatternPrivateState, IndexPatternLayer } from './types';
 
 export function updateColumnParam<
-  C extends IndexPatternColumn & { params: object },
+  C extends IndexPatternColumn, // & { params?: object },
```

Comment:
So the goal of this function is to replace the `params` property of a column, but because `params` is now optional these types are almost always `never`. The type safety of this function is probably too strict, so I'm loosening it.